styles.css

```diff
@@ -0,0 +1,30 @@
+navbar{
+    position: top;
+
+}
+header
+{
+    position: fixed;
+    width: 100%;
+}
+
+@media (max-width: 650px){
+    #ul{
+        flex-direction: column;
```

Comment:
I propose you would have first put a display of flex before applying other flex attributes